src/main/java/com/templateproject/api/controller/CommentsController.java

```diff
@@ -0,0 +1,40 @@
+package com.templateproject.api.controller;
```

Comment:
Optional : Code splitting could be optimized, you have half methods for comments inside your article controllers and half methods in this one.